_ml-commons-plugin/pretrained-models.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+title: Pretrained models
+parent: Model-serving framework
+nav_order: 120
+---
+
+# Pretrained models
+
+The model-serving framework supports a variety of open-source pretrained models that can assist with a range of machine learning (ML) search and analytics use cases. 
```

Comment:
May be we can add:

We traced a few pre-trained models in our model repo for customers so that they can use these models in opensearch.

Reply:
Addressed this by adding a caveat to the sbert link.